psc/Main.hs

```diff
@@ -78,6 +78,20 @@ compile (PSCMakeOptions inputGlob inputForeignGlob outputDir opts usePrefix) = d
             hPutStrLn stderr (P.prettyPrintMultipleWarnings (P.optionsVerboseErrors opts) warnings')
           exitSuccess
 
+globWarningOnMisses :: String -> [FilePath] -> IO [FilePath]
```

Comment:
This looks like a job for [`concatMapM`](http://haddock.stackage.org/lts-2.22/extra-1.1/Control-Monad-Extra.html#v:concatMapM). We could use the `extra` library, or just define it as a utility function here.